### PR TITLE
increase coverage

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

adding some doctests...

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning-lash/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests? [not needed for typos/docs]
- [x] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by carmocca** [comment]
Why don't we make these actual tests instead of doctests?

**Comment by Borda** [comment]
> Why don't we make these actual tests instead of doctests?

because these are also generated as examples in docs, and also useful examples in code...
not sure if you but I do not have a habit if I need to use a lib to search examples in their test instead of looking into doc or code itself :]

**Comment by Borda** [comment]
this adds almost 7% in coverage :]

File: tests/vision/test_download.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from flash.vision.embedding.model_map import _load_bolts_model
+
+
+@pytest.mark.parametrize("name", ['simclr-imagenet', 'swav-imagenet'])
+def test_load_bolts(name):
+    _load_bolts_model(name)
+
+
+def test_pull_hymenoptera(tmpdir):
+    hymenoptera_data_download(path=tmpdir)
```
Comment (tchaton) [inline tests/vision/test_download.py:26]: Let's download titanic instead which is smaller. This function `hymenoptera_data_download` can be deleted too.

**Review by tchaton — APPROVED** [review]

**Pull request merged**